datetime_exercise_2.py

```diff
@@ -0,0 +1,36 @@
+import datetime_exercise
+import radar
+
+
+def get_two_dates():
+    flag = True
+    while flag:
+        print("Enter a first date")
+        date_1 = datetime_exercise.get_date()
+        print("Enter a second date that is later than the first")
+        date_2 = datetime_exercise.get_date()
+
+        if date_1 > date_2:
+            print("First date should be earlier than the second date")
+        else:
+            flag = False
+
+    return date_1, date_2
+
+
+def main():
+    day_name = ['Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday']
+    date_1, date_2 = get_two_dates()
```

Comment:
Rename date_1, date_2 to 'start_date', 'end_date'

Reply:
start_date, end_date